__tests__/app.test.js

```diff
@@ -64,6 +64,36 @@ describe("/api/articles",()=>{
                     })
                 })
     })
+    test("GET 200 and all articles of a topic (mitch) upon request", () => {
+        return request(app)
+            .get("/api/articles?topic=mitch")
+            .expect(200)
+                .then(({body})=>{
+                    const {articles} = body
+                    expect(articles.length).toBe(12)
+                    articles.forEach((article)=>{
+                        expect(article.topic).toBe('mitch')
+                    })
+                })
+    })
+    test("GET 200 and no articles when passed a valid but non-existent topic", () => {
```

Comment:
a 404 would be more appropriate here (however this behaviour is what you would likely want to emulate in the frontend if a user tries to query a non existent topic). A 200 might be misleading to the client, as it implies that the request was successful and that the resource was found when it wasn't